service/controller/v2/key/key.go

```diff
@@ -19,6 +19,7 @@ func ClusterIDFromDrainerConfig(drainerConfig v1alpha1.DrainerConfig) string {
 
 func IsCriticalPod(podName string) bool {
 	r := false
+	r = r || strings.HasPrefix(podName, "k8s-api-healthz")
```

Comment:
I would appreciate a comment on the reasoning for this. It maybe helps people in a year when they have to work on this.

Reply:
https://github.com/giantswarm/node-operator/pull/108